node/src/actors/chain_manager/mod.rs

```diff
@@ -987,6 +995,13 @@ fn update_reputation(
     // Penalize liars and accumulate the reputation
     // The penalization depends on the number of lies from the last epoch
     let liars_and_penalize_function = liars.iter().map(|(pkh, num_lies)| {
+        if own_pkh == *pkh {
+            log::warn!(
```

Comment:
```suggestion
            log::info!(
```

I mean, losing reputation is bad but it should be logged at the same level as gaining reputation.